README.md

```diff
@@ -19,20 +19,25 @@
 
 # Apache Paimon Rust &emsp; [![Build Status]][actions] [![Latest Version]][crates.io]
 
-[Build Status]: https://img.shields.io/github/actions/workflow/status/apache-rust/paimon/ci.yml
-[actions]: https://github.com/apache-rust/paimon/actions?query=branch%3Amain
+[Build Status]: https://img.shields.io/github/actions/workflow/status/apache/paimon-rust/ci.yml
+[actions]: https://github.com/apache/paimon-rust/actions?query=branch%3Amain
```

Comment:
```suggestion
[Actions]: https://github.com/apache/paimon-rust/actions?query=branch%3Amain
```

Reply:
Hi, this label served as markdown link and not readable by users. If we do want to change this, we should change the reference too.

Reply:
I will ignore this suggestion. Feel free to reopen it if you feel like this change does needed.